plugin.yaml

```diff
@@ -81,6 +90,8 @@ dsl_definitions:
       default: false
 
   playbook_inputs: &playbook_inputs
+    ansible_playbook_path:
```

Comment:
@ahmadiesa-abu I think the name is confusing a little bit and should somehow be related the command itself because the first time I saw `ansible_playbook_path` I thought its a path to the yaml playbook